python/ray/train/tests/test_mosaic_trainer.py

```diff
@@ -0,0 +1,138 @@
+import pytest
```

Comment:
we also need to add `test_mosaic_trainer` to the `BUILD` file so that it gets run in CI!

Reply:
Done!